app/controllers/application_controller.rb

```diff
@@ -7,21 +7,48 @@ class ApplicationController < ActionController::Base
 
   before_filter :require_signin_permission!
 
-  helper_method :current_finder
+  helper_method :current_format
 
 private
 
+  def current_format
+    document_types.fetch(params.fetch(:document_type, nil), nil)
+  end
+
   def document_types
-    {
-      "cma-cases" => OpenStruct.new(
-        document_type: "cma_case",
-        title: "CMA Cases",
-      ),
+    # For each format that follows the standard naming convention, this
+    # method takes the title and name of the model class of eacg format
+    # like this:
+    #
+    # data = {
+    #   "GDS Report" => GdsReport
+    # }
+    #
+    # which will become this:
+    #
+    # {
+    #   "gds-reports" => {
+    #     klass: GdsReports
+    #     document_type: "gds-reports",
+    #     format_name: "gds_report",
+    #     title: "GDS Report"
+    #   }
+    # }
+
+    data = {
+      "CMA Case" => CmaCase
     }
-  end
 
-  def current_finder
-    document_types.fetch(params.fetch(:document_type, nil), nil)
+    data.map do |k, v|
+      {
+        k.downcase.parameterize.pluralize => OpenStruct.new(
```

Comment:
It'd be good to harden this down to an actual object if we know that `klass`, `document_type`, `format_name`, and `title` are the full interface.  [OpenStruct is problematic](http://blog.honeybadger.io/how-openstruct-and-hashes-can-kill-performance/) from a performance POV.  Of course if we're still sketching out the interface then OpenStruct is fine, but it would be good to remember to replace when we're ready.